src/wp-includes/rest-api.php

```diff
@@ -1887,13 +2144,40 @@ function rest_validate_value_from_schema( $value, $args, $param = '' ) {
  *
  * @since 4.7.0
  * @since 5.5.0 Added the `$param` parameter.
+ * @since 5.6.0 Support the "anyOf" and "oneOf" keywords.
  *
  * @param mixed  $value The value to sanitize.
  * @param array  $args  Schema array to use for sanitization.
  * @param string $param The parameter name, used in error messages.
  * @return mixed|WP_Error The sanitized value or a WP_Error instance if the value cannot be safely sanitized.
  */
 function rest_sanitize_value_from_schema( $value, $args, $param = '' ) {
+	if ( isset( $args['anyOf'] ) ) {
+		$matching_schema = rest_find_any_matching_schema( $value, $args, $param );
+		if ( is_wp_error( $matching_schema ) ) {
+			return $matching_schema;
+		}
+
+		if ( ! isset( $args['type'] ) ) {
+			$args['type'] = $matching_schema['type'];
```

Comment:
When we sanitize, I think we need to merge `$args` with the schema from `rest_find_any_matching_schema`. That way the sanitization uses all the defined keywords, not just the type and what is set in the args.

Reply:
The sanitization using the matching schema takes place right after this `if` block. It seems to me that all keywords from both parent and child schemas participate in processing already. Could you provide an example of schema which shows the problem of the current implementation?

Reply:
Ah I see on Line 2198, sorry about that. I was thinking it would fall thru. I'll do some more thinking on that.